.github/workflows/base_node_binaries.yml

```diff
@@ -24,20 +24,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-#        os: [ubuntu-18.04, ubuntu-20.04, macos-10.15, macos-11.0, windows-2019]
+#        # Broke build - cmake issue?
+#        os: [windows-2016]
 #        os: [ubuntu-latest, macos-latest, windows-latest, self-hosted]
 #        os: [ubuntu-18.04, ubuntu-20.04, macos-10.15, macos-11.0, windows-2016, windows-2019]
         os: [ubuntu-18.04, macos-10.15, windows-2019, self-hosted]
-#        os: [ubuntu-20.04]
         # https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html
 #        features: ["safe", "avx2"]
-#        features: ["safe"]
 #        target_cpu: ["x86-64", "broadwell", "skylake"]
         target_cpu: ["haswell"]
 #        target_release: ["release", "debug"]
-        exclude:
-          - os: "macos-10.15"
-            target_cpu: "haswell"
+#        exclude:
+#          - target_cpu: "x86-64"
+#            features: "avx2"
```

Comment:
There is a lot of lines that are commented out. These will just cause confusion later on. Can we remove them?

Reply:
I would prefer not, as there are reminders or options I believe we would use later. Anyone is welcome to remove them as they want too, thou I am not sure how comments can cause confusion, unless the comments are wrong or are not comments?